static.epsilon/public/epsilon_tool.json

```diff
@@ -295,31 +295,7 @@
                 ]
 
             },
-
-            {
-
-                "id": "console",
-
-                "name": "console",
-
-                "panelclass": "ConsolePanel",
-
-                "icon": "images/console.gif"
```

Comment:
Does that mean that file can be removed here, too?

Reply:
The file referred to here was already located in the educationplatform static because when direct the path used it is relative to the playground base url. The other way of referencing icons is using the cssid which are loaded relative to the tool static url.

Reply:
Thanks for clarifying.

Reply:
@szschaler  when direct the path used it is relative to the playground base url so it'll be http://127.0.0.1/8080/images/console.gif.

The other way of referencing icons is by using the css id which are loaded relative to the location of the css e.g. the tool static urls.